<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="VisibilityCheckBox">
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
     width: 21px;
     height: 21px;
 }

 QCheckBox::indicator:checked {
     image: url(:/Icons/Medium/SlicerVisible.png);
 }
QCheckBox::indicator:unchecked {
     image: url(:/Icons/Medium/SlicerInvisible.png);
 }</string>
     </property>
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="OpacityLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ctkSliderWidget" name="OpacitySlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="GroupBox_ColorBy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Color By:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="GroupBox_ColorByScalars">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Scalar Colormap</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="qMRMLColorTableComboBox" name="ColorByScalarsColorTableComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="Layout_ScalarPropertySelect">
           <item>
            <widget class="QCheckBox" name="ColorByScalarInvariantCheckBox">
             <property name="text">
              <string>Of Tensor Property</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string>ButtonGroup_ColorBySelectors</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ColorByScalarInvariantComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ScalarRangeLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Set range:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qMRMLRangeWidget" name="ColorByScalarInvariantDisplayRange">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="ColorByCellScalarsCheckBox">
           <property name="text">
            <string>Of Fiber Cluster Group ID</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string>ButtonGroup_ColorBySelectors</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Layout_SolidColorSelector">
        <item>
         <widget class="QCheckBox" name="ColorBySolidCheckBox">
          <property name="text">
           <string>Solid color</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string>ButtonGroup_ColorBySelectors</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="ctkColorPickerButton" name="ColorBySolidColorPicker"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLRangeWidget</class>
   <extends>ctkRangeWidget</extends>
   <header>qMRMLRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTractographyDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorByScalarInvariantCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ColorByScalarInvariantComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorBySolidCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ColorBySolidColorPicker</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VisibilityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpacitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ButtonGroup_ColorBySelectors"/>
 </buttongroups>
</ui>
